### ISSUE TITLE
Result browser broken when running test cases with Java 16

### ISSUE THREAD

**Issue by jowerner** [Xceptance/XLT#102]
When running test cases via mvn/surefire and Java 16, we get exceptions similar to the following:

```
java.lang.ExceptionInInitializerError
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private final java.util.Comparator java.util.TreeMap.comparator accessible: module java.base does not "opens java.util" to unnamed module @4de8b406
```

```
java.lang.NoClassDefFoundError: Could not initialize class com.thoughtworks.xstream.converters.collections.TreeMapConverter
```

This happens when creating the result browser files. XLT leaves a corrupt data.js file, which in turn breaks the result browser.

**Comment by jowerner** [comment]
XLT uses xstream to create the data.js file. xstream accesses JDK classes reflectively to ensure correct (de)serialization. Looks like certain JDK classes have been protected against manipulation via reflection in Java 16.

**Comment by jowerner** [comment]
Updated xstream to latest available version 1.4.17. In this version, xstream does no longer install each and every known (de)serializer up-front, but does so only when needed. Technically, updating the library does not fix this issue, but since XLT does not use those special (de)serializer classes at all, we should be good to go.
